src/shell/interpreter.zig

```diff
@@ -4241,10 +4301,16 @@ pub const Interpreter = struct {
                             cmd.writeFailingError("bun: ambiguous redirect: at `{s}`\n", .{@tagName(kind)});
                             return .yield;
                         }
+
+                        // Regular files are not pollable on linux
+                        const is_pollable: bool = if (bun.Environment.isLinux) false else true;
```

Comment:
Where do we check what the file type is? What makes us confident that this is always pollable on macOS? What makes us confident that this is never a path to a unix named pipe, or a character device?

Reply:
It's more of a hint, if the file descriptor is not pollable we get an error from kqueue/epoll and we don't register polls anymore